tests/js/spec/views/issueList/overview.spec.jsx

```diff
@@ -1172,23 +1172,6 @@ describe('IssueList', function () {
       wrapper = shallow(<IssueListOverview {...props} />);
     });
 
-    it('omits null values', function () {
```

Comment:
According to our types, selection values can never be null so this test isn't necessary